### PR TITLE
Have done film card component

### PR DESCRIPTION
**Pull request by Cestrus**
movie-card

### CODE REVIEW

File: src/components/FilmCardComponent/filmCard.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './filmCard.module.css';
+
+const FilmCard = () => {
+    // const { poster_path, title, release_date, genres } = props;
+    return (
+        <div className={' card ' + styles.filmCard} style={{width: 18 + 'rem'}}>
```
Comment (LordGemma) [inline src/components/FilmCardComponent/filmCard.jsx]: For concatenation strings and variables you can use template strings

**Comment by LordGemma** [comment]
Please check folder and file naming based on WiKi

File: src/components/FilmCardComponent/FilmCard.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './FilmCard.module.css';
+
+const FilmCard = () => {
+    // const { poster_path, title, release_date, genres } = props;
+    return (
+        <div className={' card ' + styles.filmCard} style={{width: 18 + 'rem'}}>
```
Comment (LordGemma) [inline src/components/FilmCardComponent/FilmCard.jsx]: use template strings instead concatenation

File: src/components/FilmCardComponent/FilmCard.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './FilmCard.module.css';
+
+const FilmCard = () => {
+    // const { poster_path, title, release_date, genres } = props;
```
Comment (JIeCoRyI3) [inline src/components/FilmCardComponent/FilmCard.jsx]: Delete unnecessary comments

File: src/components/FilmCardComponent/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import FilmCard from './filmCard';
```
Comment (LordGemma) [inline src/components/FilmCardComponent/index.js]: Currently, you have FilmCard instead of filmCard

**Pull request merged**